NangPaGo-api/NangPaGo-app/src/main/java/com/mars/app/domain/comment/community/dto/CommunityCommentResponseDto.java

```diff
@@ -7,6 +7,7 @@
 @Builder
 public record CommunityCommentResponseDto(
     Long id,
+    Long entityId,
```

Comment:
entityId 라는 이름은 안좋은것 같아요.

```suggestion
    Long postId,
```

Reply:
92c8265 반영 완료했습니다~